### ISSUE TITLE
Invalid line value on TrailingWhitespace error with gradle

### ISSUE THREAD

**Issue by bagage** [arturbosch/detekt#1082]
## Expected Behavior

When using the gradle plugin, it seems that the line number is invalid for some TrailingWhitespace error but it works as expected when executing the jar (it returns the proper line).

When using gradle, I'm expecting line 6 to be reported, similar to when using the jar.

```sh
$ ./gradlew detektCheck 
[...]
Ruleset: style - 5min debt
	TrailingWhitespace - [test.kt] at /tmp/test.kt:6:38
```

## Current Behavior

But instead, line 3 is reported:

```
$ ./gradlew detektCheck 
[...]
Ruleset: style - 5min debt
        TrailingWhitespace - [test.kt] at /tmp/test.kt:3:23
```

## Steps to Reproduce (for bugs)
Use this file:

```kt
package demo

fun main(args: Array<String>) {
    println("""
        Hello, world!
        here is a trailing space --> 
        """)
}
```

1. Run `java -jar detekt-cli-1.0.0.RC8-all.jar -i test.kt` -> line 6 is reported (OK)
2. Run `./gradlew detektCheck` -> line 3 is reported (NOK)

## Your Environment

* Version used: 1.0.0-RC8

**Comment by schalkms** [comment]
Thanks for reporting this. This issue is already fixed and will be included in the new detekt version. 
Duplicate #1030
